plugins/processors/awsappsignals/internal/attributes/attributes.go

```diff
@@ -5,38 +5,20 @@ package attributes
 
 const (
 	// aws attributes
-	AWSLocalService        = "aws.local.service"
-	AWSLocalOperation      = "aws.local.operation"
-	AWSRemoteService       = "aws.remote.service"
-	AWSRemoteOperation     = "aws.remote.operation"
-	AWSRemoteTarget        = "aws.remote.target"
-	AWSHostedInEnvironment = "aws.hostedin.environment"
+	AWSLocalService             = "aws.local.service"
+	AWSLocalOperation           = "aws.local.operation"
+	AWSRemoteService            = "aws.remote.service"
+	AWSRemoteOperation          = "aws.remote.operation"
+	AWSRemoteTarget             = "aws.remote.target"
+	AWSRemoteResourceIdentifier = "aws.remote.resource.identifier"
```

Comment:
resource.identifier and resource.type are new attributes emitted by the adot agent?

Reply:
Yes, cross-referenced the PR in the description. https://github.com/aws-observability/aws-otel-java-instrumentation/pull/786